proposals/date_shfit.json

```diff
@@ -0,0 +1,86 @@
+{
+    "id": "date_shfit",
+    "summary": "Calculates and manipulates dates and times",
+    "description": "Allows to calculate a new date and time based on a given date and time by adding or subtracting a given temporal period.",
+    "categories": [
+        "date & time"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
```

Comment:
I was not sure what to use. Most processes use data for the input value(s) and was not sure whether people would prefer the consistency or "date". Went with something generic because I was not even sure date is suitable, because a first iteration also allowed just times to be passed and now it's just datetimes basically, not just dates.

Reply:
Just saying without a strong preference: Strings currently also use "data" in text_* processes

Reply:
Yeah, I also think "text" may have been the better choice, but I think at that point where we defined that strong voices were raised for making consistent parameter names. 🤷‍♀️

Reply:
Leaving this comment unresolved for now for consideration at a later stage of the PR. Potentially switching to "date" instead of "data". By the way, I was also not sure whether to call this process calendar_shift or so...

Reply:
Will change to "date"..